mmv1/products/compute/terraform.yaml

```diff
@@ -1115,6 +1132,8 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
       candidateSubnets: !ruby/object:Overrides::Terraform::PropertyOverride
         ignore_read: true
+      adminEnabled: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
Why do we need this change?

Reply:
If I remember correctly, it's because for UPDATE operations, the provider will try to read and set the adminEnabled field for interconnects of type PARTNER_PROVIDER, and that will cause an error since adminEnabled can't be set for that case.